molecule/telemetry/telemetry-ps/tests/test_telemetry_pillar.py

```diff
@@ -778,6 +779,15 @@ def test_path_absent_after_removal(host):
                 host.check_output("yum remove -y percona-server-server")
         assert not host.file(ta_pillar_dir_ps).exists
 
+def test_ta_manually_remove_service_deb(host):
```

Comment:
I would add this only for update case based on https://perconadev.atlassian.net/browse/PKG-170 : `if update == 'yes':` . We need to keep this check for "clean install"